build_info/libkrw-dopamine0.control

```diff
@@ -0,0 +1,10 @@
+Package: libkrw-dopamine0
+Maintainer: @DEB_MAINTAINER@
+Architecture: @DEB_ARCH@
+Version: @DEB_LIBKRW_DOPAMINE_V@
+Provides: libkrw-plugin
+Conflicts: org.coolstar.libkernrw
+Replaces: org.coolstar.libkernrw
```

Comment:
Why does this conflict with kernrw?

Reply:
> Why does this conflict with kernrw?

libkernrw is currently installed on dopamine and not yet removed from 1800 dist

Reply:
Still doesn't explain why `libkrw0` conflicts with it...

Reply:
> There should also be a `Depends: firmware (>= 15.0), firmware (<= 15.4.1)`

What about 15.5 beta 1,2(maybe3?)


> Still doesn't explain why `libkrw0` conflicts with it...

Then just conflict in the plugin?

Reply:
Also, the official Dopamine release should be resetting the strap from fugu15max, so that won't matter anyways.

Reply:
> > There should also be a `Depends: firmware (>= 15.0), firmware (<= 15.4.1)`
> 
> What about 15.5 beta 1,2(maybe3?)

Then do `Depends: firmware (>= 15.0), firmware (<= 15.5)`

> > Still doesn't explain why `libkrw0` conflicts with it...
> 
> Then just conflict in the plugin?

If that.

Reply:
> Conflicts: org.coolstar.libkernrw
> Replaces: org.coolstar.libkernrw
Removed, 15.0 -> 15.5 depends done